src/Microsoft.DotNet.ProjectModel/GlobalSettings.cs

```diff
@@ -73,6 +58,41 @@ public static bool TryGetGlobalSettings(string path, out GlobalSettings globalSe
             return true;
         }
 
+        public static GlobalSettings GetGlobalSettings(Stream fs, string globalJsonPath)
+        {
+            var globalSettings = new GlobalSettings();
+
+            var reader = new StreamReader(fs);
+            JObject jobject;
+            try
+            {
+                jobject = JObject.Parse(reader.ReadToEnd());
+            }
+            catch (JsonReaderException)
+            {
+                throw new InvalidOperationException("The JSON file can't be deserialized to a JSON object.");
+            }
+
+            var projectSearchPaths = new string[] { };
+            JToken projectSearchPathsToken;
+            if (jobject.TryGetValue("projects", out projectSearchPathsToken) &&
+                projectSearchPathsToken.Type == JTokenType.Array)
```

Comment:
This seems inconsistent with PackIncludeEntry. In PIE we consider a string value to be an array with one element. Here we only consider properties whose values use array syntax. Should we make this converter more permissive and unify our story?